host/tools/build/Microsoft.Azure.Functions.DotnetIsolatedNativeHost.nuspec

```diff
@@ -17,7 +17,8 @@
     </contentFiles>
   </metadata>
   <files>
-    <file src="..\..\build\win-x64\FunctionsNetHost\*" target="contentFiles/any/any/workers/dotnet-isolated/bin" />
+    <file src="..\..\dist\windows\*" target="contentFiles/any/any/workers/dotnet-isolated/bin" />
+    <file src="..\..\dist\linux\*" target="contentFiles/any/any/workers/dotnet-isolated/bin" />
```

Comment:
This is putting both windows and linux files in the same folder. Will they not overwrite each other?

Curious why you are not using packaging from `.csproj` as well? You can control all the specific runtime stuff from there.

Reply:
The file extensions are different for windows & Linux. So we are good.

Originally we did not have this C# project (so no csporj), hence we opted for .nuspec file. Yea. We could consider switching to that. Perhaps in a follow up PR.